server/testdriver/core/src/main/java/org/infinispan/server/test/core/ContainerInfinispanServerDriver.java

```diff
@@ -600,9 +621,11 @@ public FileVisitResult postVisitDirectory(Path dir, IOException exc) {
       }
    }
 
-   static String createServerImage(String serverOutputDir) {
+   String createServerImage(String serverOutputDir, String versionToUse) {
+      String snapshotImageName = configuration.properties().getProperty(
+            TestSystemPropertyNames.INFINISPAN_TEST_SERVER_SNAPSHOT_IMAGE_NAME, SNAPSHOT_IMAGE);
```

Comment:
We should probably extract this logic to a method somewhere. It seems we're using `SNAPSHOT_IMAGE` directly in some places and ignoring the name configured via the property.

Reply:
Let me take a second look. The handler itself is cleaning these up currently, which isn't correct.